app/views/two_factor_registrations/index.html.slim

```diff
@@ -41,14 +41,14 @@
                 class: "btn btn-block btn-outline-secondary",
                 data: { "js-confirm-with-modal": "disable-totp-prompt" }
             script#disable-totp-prompt type="text/html"
-              .col-medium-center
+              .col-medium-center.text-left
```

Comment:
Here and below where `text-left` is added is further cleanup from the style refactoring not addressing existing modals properly.